source/changelogs/core-2025.5.markdown

```
@@ -1366,6 +1366,113 @@ For a summary in a more readable format:
+- Mark backflush binary sensor not supported for GS3 MP in lamarzocco ([@zweckj] - [#145406])
+- Bump yt-dlp to 2025.05.22 ([@joostlek] - [#145441])
+- Reolink fix device migration ([@starkillerOG] - [#145443])
+- Bump pysmartthings to 3.2.3 ([@joostlek] - [#145444])
+- Bump opower to 0.12.1 ([@tronikos] - [#145464])
+- Make Gemma models work in Google AI ([@tronikos] - [#145479])
+- Fix strings related to Google search tool in Google AI ([@tronikos] - [#145480])
+- Bump pyfibaro to 0.8.3 ([@rappenze] - [#145488])
+- Bump deebot-client to 13.2.1 ([@edenhaus] - [#145492])
+
+[#142933]: https://github.com/home-assistant/core/pull/142933
+[#144406]: https://github.com/home-assistant/core/pull/144406
```

markdownlint-cli2 (0.17.2)

Link and image reference definitions should be needed
Duplicate link or image reference definition: "#144406"

(MD053, link-image-reference-definitions)